### DISCUSSION TITLE
TU116-106-102

### DISCUSSION THREAD

**Discussion by Horse4Horse** [misdake/ChipAnnotationViewer#33] [Ideas]
Found almost fully made annotations for these three chips at twitter, done by @GPUsAreMagic.
Here's original [topic](https://twitter.com/gpusaremagic/status/1244303936246464517)
And jpg's [TU116](https://pbs.twimg.com/media/EUSmxq8XkAEJtnt?format=jpg&name=4096x4096), [TU106](https://pbs.twimg.com/media/EUSmyZHWsAk-N03?format=jpg&name=4096x4096), [TU102](https://pbs.twimg.com/media/EUSmzBeXkAIl2g7?format=jpg&name=4096x4096)
Hope that'll help.

**Comment by misdake** [comment]
TU104
https://misdake.github.io/ChipAnnotationViewer/?map=TU104_2&commentId=992566114
https://misdake.github.io/ChipAnnotationViewer/?map=TU104_2&commentId=992594806

**Comment by misdake** [comment]
TU106
https://misdake.github.io/ChipAnnotationViewer/?map=TU106_2&commentId=993178026

**Comment by misdake** [comment]
TU116
https://misdake.github.io/ChipAnnotationViewer/?map=TU116_2&commentId=993266390
https://misdake.github.io/ChipAnnotationViewer/?map=TU116_2&commentId=993503877

**Comment by misdake** [comment]
TU102
https://misdake.github.io/ChipAnnotationViewer/?map=TU102&commentId=994315358
